estuary/models/errata.py

```diff
@@ -47,3 +47,8 @@ class AdvisoryState(EstuaryStructuredNode):
     name = StringProperty(required=True)
     advisory = RelationshipTo('Advisory', 'STATE_OF', cardinality=ZeroOrOne)
     creator = RelationshipTo('.user.User', 'CREATED_BY', cardinality=ZeroOrOne)
+
+
+class ContainerBuildAdvisory(Advisory):
+    """Creating another label for Advisory"""
```

Comment:
Could you change this to?
"""Definition of an Errata advisory with container builds attached in Neo4j."""